### ISSUE TITLE
Component is not accessable via Keyboard. NVDA Screen reader does not read it properly

### ISSUE THREAD

**Issue by repl-soumya-c** [RSSchermer/ember-spectrum-color-picker#39]
It will be nice if this is accessable via keyboard ie Web Content Accessibility Guidelines (WCAG) 2.1 compliant. (https://www.w3.org/TR/WCAG21/ )

**Comment by RSSchermer** [comment]
Unless I'm misunderstanding you, I suspect your issue lies with with [spectrum.js](https://github.com/bgrins/spectrum), which this Ember component merely wraps.
